### ISSUE TITLE
[NPC]Tunnel Rat Geomancer

### ISSUE THREAD

**Issue by Unryzen** [Atlantiss/NetherwingBugtracker#906]
**Description**: 
Tunnel Rat Geomancer mobs in Loch Modan apply themselves a buff called [Quick Flame Ward](https://wowgaming.altervista.org/aowow/?spell=4979) . The problem is that fire spells simply break through it.

**Current behaviour**: Fire spells and fire wand damage breaks through this immunity buff as seen in the screenshot below:

https://imgur.com/a/CHERkTO

**Expected behaviour**: Fire damage doesn't supposedto break through the immunit buff.

**Server Revision**: 
2083

**Comment by Arnamik12** [comment]
are wands considered fire spells though? since that flame ward only mentions fire spells, not fire damage.

**Comment by Unryzen** [comment]
> are wands considered fire spells though? since that flame ward only mentions fire spells, not fire damage.

Yeah, I saw it only mentions spells, but wands are the only weapons that doesn't do real physical damage. They do elemental damage from one school (i.e.: fire). There are few evidence left regarding wands, but I came by [this article](http://casualwow.blogspot.com/2006/09/wands.html) what quotes the old New Players Guide and it says:

`Wand damage is reduced by resistance in the appropriate school.`

So in my reading, the same can be assumed in case of immunity.

The question is the same with engineered items like dynamites and grenades. They also do fire damage.
